### PR TITLE
Make default search limit more prominent

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closing #809

### CODE REVIEW

File: learn/getting_started/quick_start.md [hunk 1]
```diff
@@ -137,6 +137,10 @@ MeiliSearch **response**:
 }
 ```
 
+::: note
+By default, MeiliSearch returns the first **20 documents for a search query.** You can read more about the limit parameter [here](/reference/features/search_parameters.md#limit).
```
Comment (dichotommy) [inline learn/getting_started/quick_start.md:141]:
```suggestion
**By default, MeiliSearch returns only the first 20 results** for a search query. You can read more about the limit parameter [here](/reference/features/search_parameters.md#limit).
```

**Review by dichotommy — CHANGES_REQUESTED** [review]

File: learn/getting_started/quick_start.md (hunk 1 above)
Comment (dichotommy) [reply learn/getting_started/quick_start.md:141]: When picking which text to **bold**, I aim to make the bolded text readable by itself. It should be a complete clause that communicates a clear piece of information. That way someone who's scrolling quickly can still catch the pertinent information and stop to investigate if it's relevant to them.

**Review by CaroFG — APPROVED** [review]

**Review by guimachiavelli — APPROVED** [review]

**Comment by guimachiavelli** [comment]
👯

**Review by dichotommy — APPROVED** [review]
🚀

**Comment by guimachiavelli** [comment]
bors merge

**Pull request merged**